vuu-ui/packages/vuu-layout/src/flexbox/Flexbox.tsx

```diff
@@ -49,10 +45,8 @@ const Flexbox = forwardRef(function Flexbox(
     <div
       {...rest}
       className={className}
-      // data-cols={cols}
       data-resizeable={resizeable || undefined}
       id={id}
-      ref={useForkRef(rootRef, ref)}
```

Comment:
this one is needed, removing it breaks drag drop functionality. I've reverted this change and fixed the ref typing in the hook return type.